lib/packwerk/application_validator/check_package_manifests_for_privacy.rb

```diff
@@ -0,0 +1,132 @@
+# typed: strict
+# frozen_string_literal: true
+
+module Packwerk
+  class ApplicationValidator
+    class CheckPackageManifestsForPrivacy
```

Comment:
This is quite a lot of validation logic that we'll be able to extract out of packwerk!